storage/local/storage.go

```diff
@@ -336,67 +336,96 @@ func (s *memorySeriesStorage) NewPreloader() Preloader {
 	}
 }
 
-// FingerprintsForLabelMatchers implements Storage.
-func (s *memorySeriesStorage) FingerprintsForLabelMatchers(labelMatchers metric.LabelMatchers) clientmodel.Fingerprints {
+// FingerprintsForLabels implements Storage.
+func (s *memorySeriesStorage) FingerprintsForLabels(pairs ...metric.LabelPair) map[clientmodel.Fingerprint]struct{} {
```

Comment:
Actually, `FingerprintsForLabelPairs` would be the right nomenclature here if we're not going with label sets.